ui/src/shared/components/XYPlot.tsx

```diff
@@ -23,14 +23,19 @@ import {DEFAULT_LINE_COLORS} from 'src/shared/constants/graphColorPalettes'
 import {INVALID_DATA_COPY} from 'src/shared/copy/cell'
 
 // Types
-import {RemoteDataState, XYViewProperties, TimeZone} from 'src/types'
+import {
+  RemoteDataState,
+  XYViewProperties,
+  TimeZone,
+  CheckViewProperties,
+} from 'src/types'
 
 interface Props {
   table: Table
   fluxGroupKeyUnion: string[]
   loading: RemoteDataState
   timeZone: TimeZone
-  viewProperties: XYViewProperties
+  viewProperties: XYViewProperties | CheckViewProperties
```

Comment:
(Same as above... it would be great to separate this out to a different component.)

Reply:
removed!